packages/office-ui-fabric-react/src/utilities/grid/GridCell.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import {
+  autobind,
+  BaseComponent,
+  css,
+  getId,
+} from '../../Utilities';
+import { IGridCellProps } from './GridCell.Props';
+import { CommandButton } from '../../Button';
+
+export class GridCell<T, P extends IGridCellProps<T>> extends React.Component<P, {}> {
+
+  public static defaultProps = {
+    cellShape: 'circle',
+    disabled: false,
+    id: getId('gridCell')
+  };
+
+  private _id: string;
+
+  constructor(props?: P) {
+    super();
+  }
+
+  public render() {
+    let {
+      item,
+      id,
+      className,
+      role,
+      selected,
+      disabled,
+      onClick,
+      onHover,
+      onFocus,
+      onRenderItem,
+      cellDisabledStyle,
+      cellIsSelectedStyle,
+      index,
+      label
+    } = this.props;
+    return (
+      <CommandButton
+        id={ id + '-item' + index }
+        data-index={ index }
+        data-is-focusable={ true }
+        disabled={ disabled }
+        className={ css(className,
+          {
+            ['' + cellIsSelectedStyle]: selected,
+            ['' + cellDisabledStyle]: disabled
+          }
+        ) }
+        onClick={ this._onClick }
+        onMouseEnter={ this._onMouseEnter }
+        onMouseLeave={ this._onMouseLeave }
+        onFocus={ this._onFocus }
+        role={ role }
+        aria-selected={ selected }
+        ariaLabel={ label }
+        title={ label }
+      >
+        { onRenderItem(item) }
+      </CommandButton >
+    );
+  }
+
+  @autobind
+  private _onClick() {
+    let {
+        onClick,
+      disabled,
+      item
+      } = this.props as P;
```

Comment:
I think that's because one of the properties is optional right? Probably the onClick? You'd need to do a null check.